src/renderer/pages/Assets/Assets.tsx

```diff
@@ -113,14 +113,9 @@ export const Assets = () => {
         {isMultishard && (
           <div className="w-[546px] mx-auto flex items-center mt-4">
             <SmallTitleText as="h3">{t('balances.shardsTitle')} </SmallTitleText>
-            <Button
-              variant="text"
-              suffixElement={<Icon name="edit" size={16} className="text-icon-accent" />}
-              className="outline-offset-reduced"
-              onClick={toggleSelectShardsOpen}
-            >
-              {activeAccounts.length} {t('balances.shards')}
-            </Button>
+            <ButtonText icon="edit" className="outline-offset-reduced" onClick={toggleSelectShardsOpen}>
+              {`${activeAccounts.length} ${t('balances.shards')}`}
```

Comment:
Narrowed `children` to `string`, so that we always have same styles for buttons' text content
and nothing from outside will corrupt them